### ISSUE TITLE
Rethinkdb-restore fails in 2.3.0 with Python 3.4

### ISSUE THREAD

**Issue by analytik** [rethinkdb/rethinkdb#5647]
I get this error while running RethinkDB 2.3.0, python package 2.3.0 and `Python 3.4.2 (default, Oct  8 2014, 10:45:20) [GCC 4.9.1] on linux` inside a debian:jessie docker container.
```
Unzipping archive file...
Traceback (most recent call last):
  File "/usr/local/bin/rethinkdb-restore", line 9, in <module>
    load_entry_point('rethinkdb==2.3.0', 'console_scripts', 'rethinkdb-restore')()
  File "/usr/local/lib/python3.4/dist-packages/rethinkdb/_restore.py", line 266, in main
    run_rethinkdb_import(options)
  File "/usr/local/lib/python3.4/dist-packages/rethinkdb/_restore.py", line 248, in run_rethinkdb_import
    sub_dir = do_unzip(temp_dir, options)
  File "/usr/local/lib/python3.4/dist-packages/rethinkdb/_restore.py", line 150, in do_unzip
    is_fileobj = type(options["in_file"]) is file
NameError: name 'file' is not defined
```

**Comment by mlucy** [comment]
Ping @Tryneus .

**Comment by mlucy** [comment]
Thanks for the bug report @analytik !

**Comment by AlexzAK** [comment]
yep, `file` is not defined in python3

**Comment by larkost** [comment]
The fix just made it in though PR #5648 by @NeilHanlon.
